### DISCUSSION TITLE
希望能整个只录入封面的功能

### DISCUSSION THREAD

**Discussion by RingingVolcano** [the1812/Touhou-Tagger#3] [Ideas]
rt，网易云下载的会员曲是加密ncm格式 转码后其他元数据都在 但是莫得封面
而且因为没有顺序所以thtag一遍整完之后元数据也全乱了
要是能只给一张专录个封面 不动其他数据就好了

奇怪的需求x 如果方便的话还麻烦考虑下（

**Comment by the1812** [comment]
只是替换封面的话, 用 Mp3Tag 应该更合适
